React/week2 todo-app files

rajanibusani opened the pull request.

calm-prog commented on react/week2/todo-app/src/Todos/AddTodo.js:
```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+
+const AddTodo = ({ onAdd }) => {
```
Maybe I would rather name this TodoInput or something like that, so it's not misunderstood with maybe a custom hook (that contains only reusable logic). You already (rightly) name your click event handler addTodo.

calm-prog reviewed (commented):
Hi Rajani, I've just had a look at your pull request. 
This is really good work from pretty much all aspects.  You fulfilled all requirements without any doubt, but also added quite a bit to the app. You clearly know your way in programming, however, you also paid attention to the details added styles to make the application look nicer. The effort definitely paid off, it has become a really nice looking app. :+1:  Well done!

You use ES6 syntax effortlessly and you use dynamic rendering methods very well. Your code is concise and well structure. On the other hand something that took me just a little time to figure out is which component is where and what they are called. I added some comments how I think it might appear more transparent to people who read your code.

But all in all this is impressive work!

rajanibusani commented:
> Hi Rajani, I've just had a look at your pull request.
> This is really good work from pretty much all aspects. You fulfilled all requirements without any doubt, but also added quite a bit to the app. You clearly know your way in programming, however, you also paid attention to the details added styles to make the application look nicer. The effort definitely paid off, it has become a really nice looking app.  Well done!
> 
> You use ES6 syntax effortlessly and you use dynamic rendering methods very well. Your code is concise and well structured. On the other hand something that took me just a little time to figure out is which component is where and what they are called. I added some comments how I think it might appear more transparent to people who read your code.
> 
> But all in all this is impressive work!

@calm-prog A very Big THANK YOU for your time and for your comments, And thank you so much for Your words about my coding, they really gave me a bit more confidence on myself (as i came from non-it background...) and i am going to consider your comments about naming and others and will make some changes and will merge it...THANK YOU SO MUCH ONCE AGAIN...🙏 😊
